opencog/moses/main/populate_atomspace.h

```diff
@@ -56,67 +59,74 @@ void populate(AtomSpace *as, const CTable &ctable);
  * @param tnode
  * @return ValuePtr containing values such as FloatValue, LinKValues...etc
  */
-ValuePtr vertex_seq_to_value(const vertex_seq& vseq, id::type_node tnode);
+ValuePtr vertex_seq_to_value(const vertex_seq &vseq, id::type_node tnode);
+
 #define GET_ENUM [](const vertex &ver) { \
     auto en = get_enum_type(ver); \
     return ValuePtr(createNode(CONCEPT_NODE, en.getContent())); \
     }
-		
-template <typename... HandleArgs>
-inline void populate_frm_ctable(CTable _ctable, HandleArgs &&... handleArgs) {
+
+template<typename... HandleArgs>
+inline void populate_frm_ctable(CTable _ctable, HandleArgs &&... handleArgs)
+{
 	std::vector<Handle> vect1{std::forward<HandleArgs>(handleArgs)...};
 	int index = 0;
 	for (Handle h: vect1) {
 		Type type = h->get_type();
 		ValueSeq _f_values;
 		if (type == PREDICATE_NODE) {
-			boost::transform(_ctable.get_input_col_data(index), back_inserter(_f_values), bool_vertex_to_value);
+			boost::transform(_ctable.get_input_col_data(index),
+							 back_inserter(_f_values), bool_vertex_to_value);
```

Comment:
It's misaligned, basically what smart tabs would do is add tabs till it's aligned with the start of `boost::transform...` then add spaces so that `back_inserter...` is aligned with `_ctable...`. Hopefully your IDE offers indentation style.

Reply:
Okay I have added smart tabs